toolium/spec_files_loader/config_spec_files_loader.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+u"""
+Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U.
+This file is part of Toolium.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import logging
+import os
+
+from yaml import load
+
+from toolium.spec_files_loader.page_object_spec_loader import PageObjectSpecLoader
+
+
+class ConfigSpecFilesLoader(object):
+    """
+    This Loader builds all PageObject from the information given in PageObject Specification Files (YAML files).
+        - Builds all PageObjects with the given PageElements and its configuration.
+        - Build PageElements already existing in Toolium.
+        - TODO: Allow to include in the PageObject Models all PageElements implemented by the user.
+        - TODO: Allow to inherit already implemented PageObjects by the user.
+
+    Toolium Configuration required:
+        [PageObjectSpecFiles]
+        enabled: true
+        spec_files_path: resources/pageobject_spec_files
+        custom_page_object_module_path: common/pageobjects
+        custom_page_elements_module_path: common/pageelements
+    """
+
+    def __init__(self, config):
+        """
+        Configure the SpecFile Loader
+        :param config: Toolium configuration
+        """
+        self.logger = logging.getLogger(__name__)
+        self.config = config.deepcopy()
+        self.loaded_page_object_models = dict()
+        self.page_object_loader = PageObjectSpecLoader()
+
+    def load_page_object_definition(self):
+        """
+        Loads the Specification Files as YAML documents containing the definition of all PageObjects
+        and its PageElements.
+        PageObjets are not created but only the configuration is loaded.
+        """
+
+        if not self.config.getboolean_optional("PageObjectSpecFiles", "enabled", False):
+            self.logger.warn("PageObject Specification Files loader is disabled by Toolium configuration")
```

Comment:
Don't write a warn message if user is not using pageobject specification functionality